ckanext/canada/tests/test_consultations.py

```diff
@@ -82,14 +82,6 @@ def test_multiple_errors(self):
             assert err[k] == expected[k]
 
 
-    def test_example(self):
-        lc = LocalCKAN()
-        record = get_chromo('consultations')['examples']['record']
-        lc.action.datastore_upsert(
-            resource_id=self.resource_id,
-            records=[record])
-
-
```

Comment:
just to confirm, moving this method above isn't required for the tests to pass now, right?

Reply:
@wardi Correct, the order of the methods does not matter now with the change to `setup_method`

Reply:
ok, then this all looks good